frontend-react/src/hooks/filters/UseDateRange.ts

```diff
@@ -61,13 +61,13 @@ const FALLBACK_FROM_STRING = "01/01/2000";
 const FALLBACK_FROM = new Date(FALLBACK_FROM_STRING).toISOString();
 const FALLBACK_TO_STRING = "01/01/3000";
 const FALLBACK_TO = new Date(FALLBACK_TO_STRING).toISOString();
-const DEFAULT_TIME = "0:0";
+const DEFAULT_FROM_TIME = "0:0";
+const DEFAULT_TO_TIME = "23:59";
```

Comment:
Do we need to include the seconds and milliseconds to be accurate?

Reply:
Nope, this is just a default string. `date-fns` doesn't care if it's `0:0` or `00:00` for midnight.